playbooks/engineer-workflow.md

```diff
@@ -49,26 +49,38 @@ PR titles, because by default, we squash our commits when merging a PR, and this
 automatically ends up using the conventional commit format. Using the prefix in your branch name is not essential,
 but it's, well, nice.
 
-### Commits
+### Commit Messages
 
 > A well-crafted git commit message is the best way to communicate context about a change to fellow developers (and
 > indeed to [our] future selves). -[How to write a git commit message](https://chris.beams.io/posts/git-commit/)
```

Comment:
I like the quote, but this may be confusing to link to this article that doesn't follow the convention?